inference_schema/parameter_types/pandas_parameter_type.py

```diff
@@ -143,5 +144,5 @@ def input_to_swagger(self):
                 elif data_type.startswith('timedelta'):
                     swagger_schema['properties']['data']['items']['properties'][str(column_name)]['format'] = \
                         'timedelta'
-
+        swagger_schema["format"] = SWAGGER_FORMAT_CONSTANTS.PANDAS_FORMAT
```

Comment:
Should we also look at adding the orient to this format string somehow?

Reply:
that makes sense to me. does something like:

`format: pandas.DataFrame:<orient>`

look good to you?

Reply:
Sorry for the delay, yeah, that looks fine